enterprise/jmx-monitoring/src/test/java/io/crate/beans/NodeInfoTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * This file is part of a module with proprietary Enterprise Features.
+ *
+ * Licensed to Crate.io Inc. ("Crate.io") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ *
+ * Unauthorized copying of this file, via any medium is strictly prohibited.
+ *
+ * To use this file, Crate.io must have given you permission to enable and
+ * use such Enterprise Features and you must have a valid Enterprise or
+ * Subscription Agreement with Crate.io.  If you enable or use the Enterprise
+ * Features, you represent and warrant that you have a valid Enterprise or
+ * Subscription Agreement with Crate.io.  Your use of the Enterprise Features
+ * if governed by the terms and conditions of your Enterprise or Subscription
+ * Agreement with Crate.io.
+ */
+
+package io.crate.beans;
+
+import org.elasticsearch.test.ESIntegTestCase;
+import org.junit.Test;
+
+import static org.hamcrest.Matchers.isEmptyString;
+import static org.hamcrest.Matchers.not;
+
+
+public class NodeInfoTest extends ESIntegTestCase {
+
+    @Test
+    public void testNodeInfo() {
+        NodeInfo nodeInfo = new NodeInfo(() -> internalCluster().clusterService().localNode());
```

Comment:
There is no need to use a `ESIntegTestCase` just to create a `DiscoveryNode` supplier